prov/efa/src/efa_av.c

```diff
@@ -751,11 +751,6 @@ static int efa_av_remove(struct fid_av *av_fid, fi_addr_t *fi_addr,
 			break;
 		}
 
```

Comment:
Please add a comment somewhere in this function talking about how we handle operations in flight (tx and rx).

Reply:
added a comment as the functional header which explains how in flight TX and RX packets are handled.